docs/sources/trackers/web-trackers/tracker-setup/initialization-options/index.md

```diff
@@ -53,54 +55,56 @@ newTracker('sp', '{{collector_url_here}}', {
   </TabItem>
 </Tabs>
 
-The tracker will be named `sp` (tracker namespace) and will send events to the a collector url you specify by replacing `{{collector_url_here}}`. The final argument is the configuration object. Here it is just used to set the app ID and the common webPage context for each event. Each event the tracker sends will have an app ID field set to “my-app-id”.
+The tracker will send events to the Collector URL you specify by replacing `{{collector_url_here}}`. The final argument is the configuration object. Here it's used to set the app ID and the webPage entity for each event. Each event the tracker sends will have an `app_id` field set to `my-app-id`.
```

Comment:
This example sets more than just `appId` and `contexts.webPage` now, maybe this needs a revisit?